onchain/onchain.go

```diff
@@ -23,6 +23,22 @@ type Balance struct {
 	Unconfirmed uint64
 }
 
+type WalletInfo struct {
+	Id       int64
+	Name     string
+	Currency boltz.Currency
+	Readonly bool
+	EntityId *int64
+}
+
+type WalletChecker struct {
+	Id       *int64
+	Name     string
+	Currency boltz.Currency
+	Readonly bool
```

Comment:
Not sure what `Readonly` means in that context? Also allow readonly wallet? Only allow readonly?

Reply:
Its allow readonly. Can reword that